units/src/amount/unsigned.rs

```diff
@@ -110,8 +124,12 @@ impl Amount {
 
     /// Parses a decimal string as a value in the given denomination.
     ///
-    /// Note: This only parses the value string.  If you want to parse a value
-    /// with denomination, use [`FromStr`].
+    /// Note: This only parses the value string.  If you want to parse a string
```

Comment:
Not in this PR but I think we should remove all instances of double spacing. I noticed this elsewhere yesterday too.

Reply:
Opened #3761